libs/detectors/jetson/detector.py

```diff
@@ -1,3 +1,8 @@
+
+import logging
```

Comment:
Oops, these imports are not necessary, are residuals of my debugging process, you can remove them in your PR